{{ cookiecutter.repo_name }}/buildSrc/src/main/java/dependencies/dependencies.kt

```diff
@@ -25,9 +25,10 @@ object Versions {
     const val kotlin = "1.3.31"
     const val ktlint = "0.33.0"
     const val okhttp = "3.14.2"
-    const val retrofit = "2.5.0"
+    const val retrofit = "2.6.0"
```

Comment:
I think [version 2.6.1](https://github.com/square/retrofit/releases/tag/parent-2.6.1) fixes a [coroutines bug](https://jakewharton.com/exceptions-and-proxies-and-coroutines-oh-my/).